### DISCUSSION TITLE
xd-AntiSpy 4.13

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/xd-AntiSpy#75] [Announcements]
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Just released a private project: xD-AntiSpy 😎! It&#39;s a fork of the original, picking up where the last version left off. ✨ <a href="https://t.co/AfyGzHEtFw">https://t.co/AfyGzHEtFw</a><a href="https://twitter.com/hashtag/xDAntiSpy?src=hash&amp;ref_src=twsrc%5Etfw">#xDAntiSpy</a> <a href="https://twitter.com/hashtag/xpantispy?src=hash&amp;ref_src=twsrc%5Etfw">#xpantispy</a> <a href="https://twitter.com/hashtag/windows11?src=hash&amp;ref_src=twsrc%5Etfw">#windows11</a> <a href="https://twitter.com/hashtag/windows?src=hash&amp;ref_src=twsrc%5Etfw">#windows</a> <a href="https://twitter.com/hashtag/app?src=hash&amp;ref_src=twsrc%5Etfw">#app</a> <a href="https://twitter.com/hashtag/microsoft?src=hash&amp;ref_src=twsrc%5Etfw">#microsoft</a> <a href="https://twitter.com/hashtag/privacy?src=hash&amp;ref_src=twsrc%5Etfw">#privacy</a> <a href="https://t.co/wajGZtGLn5">pic.twitter.com/wajGZtGLn5</a></p>&mdash; Belim (@builtbybel) <a href="https://twitter.com/builtbybel/status/1805906464843694431?ref_src=twsrc%5Etfw">June 26, 2024</a></blockquote>

- xd-AntiSpy is now even more compact, improving performance and giving it a cleaner look due to the growing number of features. In short, xd-AntiSpy now looks more like xp-AntiSpy again
- Integrated a central full-text search. Just type your keywords directly in the dropdown or "More Tools" menu. The search resets when you clear the field or click into it
- Fixed a bug that could prevent the app from starting on Windows 10 or cause it to throw a stack trace error
- PluginButler has gained popularity and now comes bundled by default alongside the Debloater plugin
- Slight UI tweaks to the About window
- The Plugins Engine has been further optimized. Previously codenamed GothamX internally, it's finally ready to be rolled out publicly
- **Special thanks to all the translators. In less than two months, the app is now available in 10 languages!**

![explorer_hZNTrK9d31](https://github.com/user-attachments/assets/451e9cd4-bbb7-4758-8d0d-c38725a555d4)

### INFO: Today, I’ve released the almost-final version of the new "SuperMSConfig" app.  [Learn more and feel free to test the first release](https://github.com/builtbybel/SuperMSConfig)

SuperMSConfig takes the xd-AntiSpy dynamic to the next level, offering global system checks and adjustments across a wide range of areas. .  xd-AntiSpy is designed as an anti-spyware tool, while SuperMSConfig becomes the go-to hub for fine-tuning your Windows 10 or Windows 11 configuration to perfection.

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/xd-AntiSpy/releases/tag/4.13.0'>xd-AntiSpy 4.13</a>.</em>

**Comment by Pikeman66** [comment]
I just downloaded the latest zip file and gmail says it contains a virus. It’s blocked!

**Comment by blurry16** [comment]
> I just downloaded the latest zip file and gmail says it contains a virus. It’s blocked!

It doesn't. Try using a tool that doesn't specify any executable as virus, or check sources.
